app/models/concerns/generic_to_csv.rb

```diff
@@ -0,0 +1,36 @@
+require 'csv'
```

Comment:
This module is available from the climate_watch_engine gem from version 1.3.2, so we could avoid duplicating it here (unless you're making changes to it?)

Reply:
Done